### ISSUE TITLE
[dx] Add a way to download (or generate) the example datasets.

### ISSUE THREAD

**Issue by hartym** [python-bonobo/bonobo#134]
For now, tutorial needs manual download like this:

```
curl https://raw.githubusercontent.com/python-bonobo/bonobo/master/bonobo/examples/datasets/coffeeshops.txt > `python -c 'import bonobo; print(bonobo.get_examples_path("datasets/coffeeshops.txt"))'`
```

This is not the best, and although it's not desirable to bundle the datasets within the eggs/wheels (which would add unnecessary files to the packaging), there should be an easy way to download the datasets. Github does not allow `git archive`, so probably we should add a command that download the missing files.

`bonobo examples download` ??? Just suggesting, not thought about it.

**Comment by hartym** [comment]
It's even worse on windows, where the shell syntax may completely differ. We NEED a way to overcome that.

**Comment by hartym** [comment]
There is also a problem when we use "system" python, installed as root, because user won't have the rights to install the example datasets. We need to download it in userland, maybe ?

**Comment by hartym** [comment]
As it is not easy at all to download files from github (I did implement it but it needs a lot of API requests, it is slow, and you will eventually be rate limited by github really quick if no authentication is provided, which is not something trivial to do, especially for such a small use case).

Let's package it differently: it can be either a S3 bucket, maybe a different git repository, or some other kind of archive we download somewhere.

**Comment by hartym** [comment]
Will be released in 0.6.
